src/main/scala/edu/ie3/osmogrid/lv/coordinator/LvCoordinator.scala

```diff
@@ -264,6 +268,11 @@ object LvCoordinator extends ActorStopSupportStateless {
                 assetInformation
               )
             ) =>
+          // building a complete street graph
+          val (highways, highwayNodes) =
+            OsmoGridModel.filterForWays(osmoGridModel.highways)
+          streetGraph = buildStreetGraph(highways.seq.toSeq, highwayNodes)
```

Comment:
The `LvGridGenerator` can't build the street graph, because it will only receive a small part of the osm data.
An option would be to build the street graph in the `LvCoordinator` and the send the part needed to the `LvGridGenerator`. In this way we will build the complete street graph only once.